apps/emqx_conf/src/emqx_conf.erl

```diff
@@ -314,18 +316,17 @@ gen_flat_doc(RootNames, #{full_name := FullName, fields := Fields} = S) ->
         text => short_name(FullName),
         hash => format_hash(FullName),
         doc => maps:get(desc, S, <<"">>),
-        fields => format_fields(Fields)
+        fields => format_fields(Fields, DescResolver)
     }.
 
-format_fields([]) ->
-    [];
-format_fields([Field | Fields]) ->
-    [format_field(Field) | format_fields(Fields)].
+format_fields(Fields, DescResolver) ->
+    [format_field(F, DescResolver) || F <- Fields].
 
-format_field(#{name := Name, aliases := Aliases, type := Type} = F) ->
+format_field(#{name := Name, aliases := Aliases, type := Type} = F, DescResolver) ->
     L = [
         {text, Name},
         {type, format_type(Type)},
+        {typedoc, format_type_desc(Type, DescResolver)},
```

Comment:
For this version, maybe we can append `dos` after `typedoc`  (insert a `<br/>` in between)

Reply:
i mean, don't introduce a new field.

Reply:
I was under impression that such manipulations should be under the docsite team responsibility.